hyperapp.d.ts

```diff
@@ -0,0 +1,45 @@
+export as namespace hyperapp
+
+// VNode part
+
+export interface VNode<Data> {
+  tag: string
+  data?: Data
+  children: VElement<{}>[]
+}
+
+export type VElement<Data> = VNode<Data> | string
+
+export interface Component<Data> {
+  (data?: Data, ...children: VElement<{}>[]): VNode<Data>
+}
+
+export function h<Data>(
+  tag: Component<Data> | string,
+  data?: Data,
+  children?: VElement<{}>[]
+): VNode<Data>
+export function h<Data>(
+  tag: Component<Data> | string,
+  data?: Data,
+  ...children: VElement<{}>[]
+): VNode<Data>
+
```

Comment:
@Swizz Is there any way to declare that using the same definition? If it's impossible, then can we leave it out? The fact that children can be `arguments` is an implementation detail needed by JSX, but users should definitely not use `h` like that.